i18n/__init__.py

```diff
@@ -45,8 +46,6 @@ def __call__(self, **kwargs):
         args = self.parser.parse_known_args(self.args)[0]
```

Comment:
Now that config.py doesn't create a Configuration, maybe we can move the "from . import config" line just above to the module level?